README.md

```diff
@@ -210,3 +211,261 @@ developer console](https://code.google.com/apis/console/).
 ### Format of the payload
 
 This is the same format as [the reminder from the reminders API](#get-remindersid).
+
+## API users
+
+This API manages users.
+
+### POST `/users`
+
+This creates a new user.
+
+#### Input
+
+All properties must be present. There is no default value. For example:
+```json
+{
+  "forename": "Sherlock",
+  "email": "Sherlock.Holmes@baker-street.co.uk",
+  "password": "MoriartyShallReturn"
+}
+```
+
+Note that the password is stored in database using a modern hash algorithm
+(Argon2 currently).
+
+Currently the user is created as-is. In the future we'll implement a 2-step
+creation mechanism where the email would be used.
+
+#### Output
+* 201 if request succeeded, with `location` header indicating the new resource URL.
+* 400 if some properties are missing or invalid.
+
+### GET `/users/{id}`
+
+Required: the header `Authorization` that identifies a user. Only users
+belonging in the same group(s) can access information about another user.
+
+This returns information about a user.
+
+#### Output
+* 200 if request succeeded, with body:
+```json
+{
+  "forename": "Sherlock",
+  "email": "Sherlock.Holmes@baker-street.co.uk"
```

Comment:
Do we need to return the email address?  Seems like it would be preferable to try not to leak this, and I'm not sure what it'll be used for after this request.